.expeditor/config.yml

```diff
@@ -3,21 +3,32 @@
 github:
   # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
   # is bumped automatically via the `built_in:bump_version` merge_action.
-  version_file: "VERSION"
+  # TODO: Change this to `VERSION` once we are live
+  version_file: "VERSION_FAKE"
+  version_tag_format: 'fake-{{version}}'
   # The file where our CHANGELOG is kept. This file is updated automatically with
   # details from the Pull Request via the `built_in:update_changelog` merge_action.
-  changelog_file: "CHANGELOG.md"
+  changelog_file: "CHANGELOG_FAKE.md"
   delete_branch_on_merge: true
+  release_branch:
+    - master:
+        # Version constraint at this point is arbitrary but represents 1.0.0
+        version_constraint: 1*
 
 # Slack channel in Chef Software slack to send notifications about Expeditor actions
 slack:
   notify_channel: habitat-notify
 
 # These actions are taken, in order they are specified, anytime a Pull Request is merged.
 merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Expeditor: Skip Version Bump"
+        - "Expeditor: Skip All"
```

Comment:
We'll need these added to the repository.

Reply:
Done!